expression/src/error.rs

```diff
@@ -35,6 +35,11 @@ pub enum ExpressionExecutionError {
     #[snafu(display("Cannot load expression for execution"))]
     LinkExpression { source: libloading::Error },
 
+    #[snafu(display(
+        "Error during compilation of expression. Stdout: {stdout}, Stderr: {stderr}"
```

Comment:
```suggestion
        "Error during compilation of expression."
```

Nicht an den User, oder?